style.css

```
@@ -0,0 +1,71 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    min-height: 100vh;
+    background: #d3d3d3;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+::-webkit-scrollbar{
+    display: none;
+}
+
+.paper{
+    position: relative;
+    background: rgba(190, 204, 208, 0.944);
+    width: 90%;
+    max-width: 800px;
+    min-width: 400px;
+    height: 480px;
+    margin: 0 auto;
+    border-radius: 5px;
+    box-shadow: 5px 5px 20px 5px black;
+    overflow: hidden;
+}
+
+.paper::before{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    width: 60px;
+    background: radial-gradient(grey 6px, transparent 7px)repeat-y;
+    background-size: 30px 30px;
+    border-right: 3px solid green;
+    box-sizing: border-box;
+}
+
+.paper-content{
+    position: absolute;
+    top: 30px;
+    right: 0;
+    bottom: 30px;
+    left: 60px;
```

```suggestion
    inset: 30px 0 30px 60px;
```
You should use shorthand properties instead of writing all of it....

---

i just updated it as shorthand properties..